crown-court-contribution/src/test/java/uk/gov/justice/laa/crime/contribution/controller/ContributionControllerContributionIntegrationTest.java

```diff
@@ -293,10 +292,10 @@ void givenApiMaatCalculateContributionRequest_whenCheckContributionRuleIsInvoked
 
     @Test
     void givenApiMaatCalculateContributionRequest_whenCheckContributionRuleIsInvoked_thenFalseIsReturned() throws Exception {
-        ApiMaatCalculateContributionRequest calculateContributionRequest =
-                TestModelDataBuilder.buildCalculateContributionRequest();
-        calculateContributionRequest.setCrownCourtOutcome(List.of(getApiCrownCourtOutcome(CrownCourtOutcome.AQUITTED)));
-        String requestData = objectMapper.writeValueAsString(calculateContributionRequest);
+        ApiMaatCheckContributionRuleRequest apiMaatCheckContributionRuleRequest =
+                TestModelDataBuilder.buildCheckContributionRuleRequest();
+        apiMaatCheckContributionRuleRequest.setCrownCourtOutcome(List.of(getApiCrownCourtOutcome(CrownCourtOutcome.AQUITTED)));
+        String requestData = objectMapper.writeValueAsString(apiMaatCheckContributionRuleRequest);
         mvc.perform(buildRequestGivenContent(HttpMethod.POST, requestData, CHECK_CONTRIBUTION_RULE_ENDPOINT_URL))
                 .andExpect(status().isOk())
```

Comment:
I think we should also check the boolean response object here too.

Reply:
Already checking the boolean response at line 302